### ISSUE TITLE
android app switching

### ISSUE THREAD

**Issue by thisajh** [nesbox/TIC-80#734]
on the android version, when i switch to home screen or another app and switch back to tic80 the app shows only a black screen and wont respond to inputs.  resulting in a loss of any unsaved work.

**Comment by carnivion** [comment]
Yes, I have the same problem. Please fix this critical problem first!
